packages/types/src/fulu/sszTypes.ts

```diff
@@ -125,6 +126,14 @@ export const BeaconState = new ContainerType(
   {typeName: "BeaconState", jsonCase: "eth2"}
 );
 
+export const DAContents = new ContainerType(
```

Comment:
what about just calling it `DataContents`?

Using "DA" here is not ideal because in Fulu we use columns as DA mechanism, not blobs.

generally, dislike abbreviations

Reply:
Yeah, its a little confusing when/where cells come into play.

EL->BN->VC->BN  still passes blobs for the entire block proposal (production and publish) flow.

In between, BN sanity checks proofs (which are cell proofs) by reconstructing cells. And during publishing, cells are reconstructed. We cache these produced cells to avoid reconstructing twice.

Reply:
Turns out its not even needed. bye bye `Contents`